src/full/Agda/TypeChecking/Substitute.hs

```diff
@@ -1427,34 +1427,38 @@ instance (Subst t a, Ord a) => Ord (Elim' a) where
 ---------------------------------------------------------------------------
 
 -- | @univSort' univInf s@ gets the next higher sort of @s@, if it is
---   known (i.e. it is not just @UnivSort s@). @univInf@ is returned
---   as the sort of @Inf@.
+--   known (i.e. it is not just @UnivSort s@).
 --
 --   Precondition: @s@ is reduced
-univSort' :: Maybe Sort -> Sort -> Maybe Sort
-univSort' univInf (Type l) = Just $ Type $ levelSuc l
-univSort' univInf (Prop l) = Just $ Type $ levelSuc l
-univSort' univInf Inf      = univInf
-univSort' univInf s        = Nothing
-
-univSort :: Maybe Sort -> Sort -> Sort
-univSort univInf s = fromMaybe (UnivSort s) $ univSort' univInf s
-
-univInf :: (HasOptions m) => m (Maybe Sort)
-univInf =
-  ifM ((optOmegaInOmega <$> pragmaOptions) `or2M` typeInType)
-  {-then-} (return $ Just Inf)
-  {-else-} (return Nothing)
+univSort' :: Sort -> Maybe Sort
```

Comment:
Consider renaming to `succSort'`.  Reading through the PR I always wondered what `univSort` meant.  

But then, the resp. constructor should be renamed as well.

Reply:
Or maybe `sucSort'`? `suc`seems to be the standard notation for the successor in Agda.

Reply:
This doesn't really have anything to do with this PR, feel free to file a separate issue if you want to rename `univSort` to something else. I actually prefer it over `succSort` since it is more descriptive: it is the sort of a universe.